internal/kafka/confluent_kafka.go

```diff
@@ -243,6 +244,10 @@ func consumeMessage(message *ckafka.Message, h *GroupHandler) error {
 		jsonMessage = fmt.Sprintf("Timestamp: %d\t%s", message.Timestamp.UnixMilli(), jsonMessage)
 	}
 
+	if h.Properties.PrintOffset {
+		jsonMessage = fmt.Sprintf("[Partition:Offset]: [%d:%s]\t%s", message.TopicPartition.Partition, message.TopicPartition.Offset.String(), jsonMessage)
```

Comment:
I assume we print this for messages other than JSON messages? Can we fix the variable name here?

Reply:
json message is concatenated after partition:offset. suggestion for a var name?

Reply:
What about `message`? It seems like the message could be either JSON or non-JSON here.

Reply:
that's the "kafka.message". I changed this to `messageString`